cmd/fuseml_core/main.go

```diff
@@ -50,21 +51,25 @@ func main() {
 	var (
 		runnableSvc runnable.Service
 		codesetSvc  codeset.Service
+		workflowSvc workflow.Service
 	)
 	{
 		runnableSvc = svc.NewRunnableService(logger, core.NewRunnableStore())
 		codesetSvc = svc.NewCodesetService(logger, core.NewGitCodesetStore(gitAdmin))
+		workflowSvc = svc.NewWorkflow(logger, core.NewWorkflowStore())
```

Comment:
Could you use `NewWorkflowService` to be consistent with the other names ?

Reply:
Done. Thanks, I completely missed it.